packages/react-components/src/icons/orcid-social.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+/* istanbul ignore file */
+
+const orcidSocialIcon = (
```

Comment:
Meh. Can we use the other one (seems better aligned) and override its default green color maybe?

Reply:
I spoke to Beatriz about doing this but she prefered having two icons

Reply:
Okay